compiler/luci/pass/src/PropagateQParamBackwardPass.cpp

```diff
@@ -15,11 +15,201 @@
  */
 
 #include "luci/Pass/PropagateQParamBackwardPass.h"
+#include "QuantizationUtils.h"
 
 #include <luci/IR/CircleNodes.h>
 #include <luci/IR/CircleNodeVisitor.h>
+#include <luci/Service/Nodes/CircleConst.h>
 #include <luci/Log.h>
 
+#include <cmath>
+
+namespace
+{
+
+void quant_const_values(luci::CircleConst *const_node, float scaling_factor, float zerop,
+                        loco::DataType quant_type)
+{
+  uint32_t size = const_node->size<loco::DataType::FLOAT32>();
+
+  const float scaling_factor_inv = 1.0 / scaling_factor;
```

Comment:
(optional) I PR looks relocation so maybe not related with this PR but is `scaling_factor` checked with zero value?

Reply:
Hmm.. it is not checked anywhere. We may have to throw an exception in that case.